modules/cmake_configure_file/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/wep21/cmake_configure_file",
```

Comment:
Not in this PR, but can you add a README.md file to this repository? Otherwise, it's not obvious what this module is for.